src/components/Footer.jsx

```diff
@@ -1,7 +1,40 @@
 import React from "react";
+import {
+  FaInstagram,
+  FaEnvelope,
+  FaSquareFacebook,
+  FaLinkedin,
+} from "react-icons/fa6";
```

Comment:
After you use map, remove these icons from import, because you already did when you imported the array of links/icons.